config/routes.rb

```diff
@@ -13,6 +15,10 @@
   resources :relationships do
     post :upload, on: :collection
   end
+  resources :source_titles
   resources :user_projects
   resources :users
+  resources :works
+
+  get 'names', to: 'names#index'
```

Comment:
For consistency, let's set this up as a `resources` statement as well, only allowing the `index` route.